### DISCUSSION TITLE
Can I cancel my current claimed domain and claim another domain?

### DISCUSSION THREAD

**Discussion by Mr-vabs** [community/community#36725] [GitHub Education]
Hello 👋,

I've claimed for example abcd.me and abcd.tech domain from benefits but now I want diffrent domain so can I cancel these domains and claim for other with the same .me and .tech for example new.me and new.tech like?

Or I'll list both of the claimed and can't claim a new one too? 

***Please help me!***

**Comment by ossanna16** [comment]
Hi @Mr-vabs,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on web development. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Mr-vabs** [reply]
The domain of which I'm taking about, got with GitHub student developer pack that's why I've asked here.

**Reply by ossanna16** [reply]
Thank you for clarifying!
